### ISSUE TITLE
关于论文公式（2）的理解问题

### ISSUE THREAD

**Issue by yyr6661** [summerlvsong/Aggregation-Cross-Entropy#24]
对general loss function为何可以通过第3节的公式（2）来估计？估计公式中的每一项概率不是远大于general loss function中的对应项吗？
